templates/firefox-ios.xliff

```diff
@@ -3581,7 +3581,7 @@ Private Tab</source>
         <note>After engaging with the microsurvey prompt, the microsurvey pops up as a bottom sheet for the user to answer, this is the title for the header on the screen.</note>
       </trans-unit>
       <trans-unit id="Microsurvey.Survey.Options.LikertScaleOption1.v127" xml:space="preserve">
-        <source>Very Satisfied</source>
+        <source>Very satisfied</source>
```

Comment:
(This is OK as it's "just" a capitalization change and localizers deal with that accordingly, so no need to edit the string ID)